packages/ra-ui-materialui/src/input/NullableBooleanInput.tsx

```diff
@@ -38,7 +40,7 @@ const NullableBooleanInput = ({
     validate,
     ...rest
 }) => {
-    const classes = useStyles();
+    const classes = useStyles({});
```

Comment:
Just curiosity, but why is the impact of the brackets as arguments of `useStyles`?

Reply:
None. See https://github.com/mui-org/material-ui/issues/16867

Reply:
In a nutshell, it's a workaround for TypeScript